### ISSUE TITLE
Burial | Accessibility Testing

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#77400]
## Issue Description
_To prepare for Staging Review and launch, the Burial 21-530EZ form needs to be tested for accessibility based on the requirements outlined in #77395._

Results should be documented in [this spreadsheet](https://docs.google.com/spreadsheets/d/1VRlaHupJVuAnxUJTohFp-v75Z0dDSqR5z6gPuMm9udA/edit?usp=sharing).

---

## Tests to Complete (see specifics in #77395)
Note: These tests need to be completed for authenticated and unauthenticated users.
- [x] Use of color and color contrast checks
- [x] axe checks
- [x] axe DevTools scan results
- [x] How did axe testing go?
- [x] Content zoom and reflow checks
- [x] Keyboard navigation checks

Note: We can only address non-platform accessibility issues.

## Acceptance Criteria
- [x] All required accessibility checks in #77395 have been completed and documented in [this spreadsheet](https://docs.google.com/spreadsheets/d/1d6McDhyQBAWeJt08a1KRmsGoQiBKUoOY1JX7whYcmVw/edit?usp=sharing)
- [ ] #77395 submitted to Platform Team
---

**Comment by steele-lm** [comment]
For awareness...@fiorella-io and @jstrothman
